server/webapp/WEB-INF/rails/webpack/views/components/table/index.tsx

```diff
@@ -137,4 +211,9 @@ export class Table extends MithrilViewComponent<Attrs> {
   private static unspecifiedValue() {
     return ("");
   }
+
+  private static disableEvent(e: MouseEvent) {
```

Comment:
Please, can you also update the PR description of why this event needs to be disabled?

Reply:
Done